data/json/itemgroups/ammo.json

```diff
@@ -31,21 +31,23 @@
     ]
   },
   {
-    "type" : "item_group",
-    "id" : "ammo_pistol_milspec",
-    "//" : "Military specification pistol ammo found at military sites.",
+    "type": "item_group",
+    "id": "ammo_pistol_milspec",
+    "//": "Military specification pistol ammo found at military sites.",
     "items": [
        [ "46mm", 70 ],
+       [ "5x50dart", 50 ],
```

Comment:
Too frequent when compared to 9mm. Consider 20 for the regular and 10 for the heavy.